webpack.config.js

```diff
@@ -12,7 +12,8 @@ var wpconfig = {
     context: __dirname,
 
     entry: {
-        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js'
+        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js',
```

Comment:
I find it a little strange to have a mix of Studio and LMS configurations in one file. What would it take to have a config in each root directory, i.e. "lms" and "cms"? Is there a benefit to keeping a central file? I can see some benefit to a standard way of importing common libraries like the UI Toolkit.